homeassistant/components/skybell/const.py

```diff
@@ -0,0 +1,21 @@
+"""Constants for the Skybell HD Doorbell."""
+from datetime import timedelta
+from typing import Final
+
+DATA_COORDINATOR = "coordinator"
+DATA_DEVICES = "devices"
+
+DOMAIN: Final = "skybell"
+DEFAULT_NAME = "SkyBell"
+
+ATTRIBUTION = "Data provided by Skybell.com"
+
+NOTIFICATION_ID = "skybell_notification"
+NOTIFICATION_TITLE = "Skybell Sensor Setup"
+
+DEFAULT_CACHEDB = "./skybell_cache.pickle"
```

Comment:
It looks like this will break if there are multiple entries

Reply:
Investigating. Hoping @kevchu3 can send me invite for testing.